tools/import-image-tests

```diff
@@ -147,15 +157,43 @@ def main():
     if args.db_ignore:
         with open(args.db_ignore, "r", encoding="utf-8") as f:
             db_ignore = f.read().splitlines()
+        print("Filtering manifests, ignoring:")
```

Comment:
I'd add the print to where the actual ignore takes place (line 190).

Reply:
That might be better indeed.